### PR TITLE
Model plugin deployer fix

### PR DESCRIPTION
**Pull request by colin-grapl**
### Which issue does this PR correspond to?
This is a hotfix for a service that is still MVP.

### What changes does this PR make to Grapl? Why?
This runs part of the deployment process in a background thread. This is really an awful workaround for the fact that the deployment isn't transactional.

### How were these changes tested?

Tested locally and in AWS.

### CODE REVIEW

**Comment by wimax-grapl** [comment]
i'm assuming you want staging for this not master

**Comment by wimax-grapl** [comment]
did the change on colin's behalf

File: src/python/grapl-model-plugin-deployer/src/grapl_model_plugin_deployer.py [hunk 1]
```diff
@@ -470,15 +471,17 @@ def upload_plugins(s3_client, plugin_files: Dict[str, str]) -> Optional[Response
...
         raw_schemas,
-    )
+    ))
+    th.start()
 
     for path, file in plugin_files.items():
         upload_resp = upload_plugin(s3_client, path, file)
         if upload_resp:
             return upload_resp
+    th.join()
```
Comment (wimax-grapl) [inline src/python/grapl-model-plugin-deployer/src/grapl_model_plugin_deployer.py:484]: does this thread get auto joined if we ever `return upload_resp` or does it just spin forever

Comment (colin-grapl) [reply src/python/grapl-model-plugin-deployer/src/grapl_model_plugin_deployer.py:484]: it'll die eventually, but yeah probably worth a finally or something

**Review by jgrillo-grapl — APPROVED** [review]

**Pull request merged**